### PR TITLE
Debase ActiveMQ container image from intermediate base images

### PR DESCRIPTION
**Pull request by fabricetriboix**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,13 +24,18 @@ ENV ACTIVEMQ="apache-activemq-$ACTIVEMQ_VERSION" \
...
 
 WORKDIR /app
 COPY activemqrc /app/home/.activemqrc
 COPY jmx-prometheus-config.yaml .
 
-RUN curl -fsSLo activemq.tgz "https://archive.apache.org/dist/activemq/${ACTIVEMQ_VERSION}/${ACTIVEMQ}-bin.tar.gz" \
+RUN     yum -y update \
+        && yum -y install java-11-openjdk \
+        && yum clean all \
+        && curl -fsSLo activemq.tgz "https://archive.apache.org/dist/activemq/${ACTIVEMQ_VERSION}/${ACTIVEMQ}-bin.tar.gz" \
```
Comment (JonHolman) [inline Dockerfile]: Can this tar file be copied to our s3 bucket for artifacts so we don't need to download it over the internet?

Comment (fabricetriboix) [reply Dockerfile]: Yes, good point @JonHolman , I will do that

Comment (JonHolman) [reply Dockerfile]: @fabricetriboix  since you added the script now, this line should change.

**Review by JonHolman — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (fabricetriboix) [reply Dockerfile]: @JonHolman  Apologies, I forgot to commit my latest Dockerfile... I will do it now.

File: Dockerfile [hunk 2]
```diff
@@ -30,21 +31,18 @@ ENV ACTIVEMQ="apache-activemq-$ACTIVEMQ_VERSION" \
 
 WORKDIR /app
 COPY activemqrc /app/home/.activemqrc
-COPY jmx-prometheus-config.yaml .
+COPY jmx-prometheus-config.yaml \
+    "artifacts/${ACTIVEMQ}-bin.tar.gz" \
+    "artifacts/${JMX_PROMETHEUS_AGENT}" \
```
Comment (JonHolman) [inline Dockerfile]: Please be consistent with the way you're creating the filenames, using ENVs vs concatenating them in place.  ACTIVEMQ is concatenating in place and JMX_PROMETHEUS_AGENT is using an ENV.  Compare a find on "JMX_PROMETHEUS_AGENT" to a find on ${ACTIVEMQ}-bin.tar.gz

**Review by JonHolman — APPROVED** [review]

**Pull request merged**